src/Lib/App/Error.hs

```diff
@@ -9,22 +9,28 @@ module Lib.App.Error
 
 import Control.Monad.Except (MonadError, throwError)
```

Comment:
You also need to add this import to make module compile:

https://github.com/Holmusk/CDMP/blob/cd24a6fb3f3d187f68bdd0d8ff7056ba456f5f58/backend/src/CDMP/App/Error.hs#L47